api/models/user_permission.py

```diff
@@ -0,0 +1,35 @@
+from api.db import db
+
+
+class UserPermission(db.Model):
+    """
+    Manage Permission of Users
+    """
+    __tablename__ = 'user_permissions'
+
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String, unique=True)
+    description = db.Column(db.String)
+    verified_user = db.Column(db.Boolean)
```

Comment:
User saved in the DB is verified user as we are sending verification mail so the proper logic can be setup in the frontend and no need of this here? Use case for providing it here as firebase already has the metadata for the same